app/admin/users.rb

```diff
@@ -9,21 +9,30 @@
     column :email
     column :created_at
     actions
-  end
-  
-  filter :first_name
-  filter :last_name
-  filter :email
-  filter :created_at
+  end  
   
   form do |f|
     f.inputs do      
       f.input :first_name
       f.input :last_name
       f.input :email
-      f.input :password
-      f.input :password_confirmation
     end
     f.actions
   end
+
+  controller do
+    def create
+      params[:user][:password] = Devise.friendly_token[6, 10]
+      @user = User.new(permitted_params[:user])
+      #@user.skip_confirmation_notification! # Disable confirmation email notification
+      super
+    end
+
+    def update
+      # Disable confirmation email notification
+      #@user = User.find(params[:id])
+      #@user.skip_confirmation_notification!
+      super
+    end
```

Comment:
Since no actions are being done here you can remove the method completely and it will default to the parent class (`super`).